appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java

```diff
@@ -161,11 +161,13 @@ public VmApiProxyDelegate() {
     try {
       byte responseData[] = runSyncCall(environment, packageName, methodName, requestData, timeoutMs);
       long end = System.currentTimeMillis();
-      logger.log(Level.INFO, String.format(
+      if (logger.isLoggable(Level.FINE)) {
+        logger.log(Level.FINE, String.format(
           "Service bridge API call to package: %s, call: %s, of size: %s " +
-          "complete. Service bridge status code: %s; response " +
-          "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
+            "complete. Service bridge status code: %s; response " +
```

Comment:
No need to change as it is consistent with the file. So comment is more of an FYI about the line wrapping which should come before the '+' (https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break). I am surprised that the formatter didn't fix that for you.